package/MDAnalysis/core/groups.py

```diff
@@ -1359,6 +1394,87 @@ class AtomGroup(GroupBase):
         new :class:`AtomGroup` for multiple matches.  This makes it
         difficult to use the feature consistently in scripts.
 
+    AtomGroups can be compared and combined using group operators. For
+    instance, AtomGroups can be concatenated::
+
+        ag_concat = ag1 + ag2  # or ag_concat = ag1.concat(ag2)
```

Comment:
I wouldn't put the alternative in a comment. Either a separate code line or mention it in text with a link to `concat()`.